plugins/redux/src/redux/istore.d.ts

```diff
@@ -0,0 +1,5 @@
+import icounter from "@redux/reducers/counter/counter";
+
+export interface istore {
```

Comment:
```suggestion
export interface IStore {
```
Bence interface büyük harfle başlamalı